sdks/python/apache_beam/runners/dataflow/dataflow_runner_test.py

```diff
@@ -863,6 +863,31 @@ def test_pack_combiners_enabled_by_experiment(self):
     self._test_pack_combiners(
         PipelineOptions(self.default_properties), expect_packed=True)
 
+  def test_resource_hints_translation(self):
+    runner = DataflowRunner()
+
+    self.default_properties.append('--experiments=use_legacy_bq_sink')
+    self.default_properties.append('--resource_hint=min_ram_per_vcpu=10GB')
+    with beam.Pipeline(runner=runner,
+                       options=PipelineOptions(self.default_properties)) as p:
+      # pylint: disable=expression-not-assigned
+      (
+          p
+          | beam.Create([1])
+          | 'MapWithHints' >> beam.Map(lambda x: x + 1).with_resource_hints(
+              min_ram_per_vcpu='20GB',
+              accelerator='type:nvidia-tesla-k80;count:1;install-nvidia-drivers'
+          ))
+
+    step = self._find_step(runner.job, 'MapWithHints')
+    self.assertEqual(
+        step['properties']['resource_hints'],
+        {
+            'beam:resources:min_ram_per_vcpu_bytes:v1': '10000000000',
```

Comment:
FWIW, memory is almost always GiB, even when written GB. THis is the convention for GCE too, so we should probably follow that. https://cloud.google.com/compute/docs/instances/creating-instance-with-custom-machine-type

Reply:
Discussed offline, keeping as is for now.